### PR TITLE
Implemented a compressing store

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: tests/Stores/CachingStoreTest.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
  * file that was distributed with this source code.
  */
 
-namespace StyleCI\Tests\Storage\Throttlers;
```
Comment (joecohens) [inline tests/Stores/CachingStoreTest.php]: lol

**Comment by joecohens** [comment]
:heart_eyes:

File: tests/Stores/CachingStoreTest.php (hunk 1 above)
Comment (GrahamCampbell) [inline tests/Stores/CachingStoreTest.php]: lol - i copy and paste file headers and imports all the time :P

Comment (GrahamCampbell) [inline tests/Stores/CachingStoreTest.php]: sometimes i make a mess of it, lol

**Comment by GrahamCampbell** [comment]
:ship:

**Pull request merged**